python/ray/_private/services.py

```diff
@@ -1465,6 +1467,7 @@ def start_raylet(redis_address,
         f"--redis-address={redis_address}",
         f"--temp-dir={temp_dir}",
         f"--metrics-agent-port={metrics_agent_port}",
+        f"--tracing-startup-hook={tracing_startup_hook}",
```

Comment:
don't think we need to pass this flag to worker anymore because we are using internal kv now.